addons/disable-sprite-wobble/userstyle.css

```diff
@@ -0,0 +1,7 @@
+[class*="sprite-selector_raised"]:hover {
+  -webkit-animation-name: none;
+  animation-name: none;
+  -webkit-transform: none;
+  -ms-transform: none;
+  transform: none;
```

Comment:
Remove the prefixed properties

Reply:
Done ✅

Reply:
I copied these properties from scratch btw

Reply:
That's true. But Scratch Addons is only used on Firefox and Chromium browsers, so it shouldn't be much of an issue.

Reply:
Webkit is chromium